### DISCUSSION TITLE
Adding co-supervisor

### DISCUSSION THREAD

**Discussion by madhavjagan91** [wang-chen/thesis_template_ntu#13] [Q&A]
Hi Wang-Chen. Thank you for making this template. Its a huge help for our students.

I have a question on how to add co-supervisor in the template. Because I have 2 supervisors

**Comment by wang-chen** [comment]
I remember that only your main supervisor is supposed to sign on the thesis, can you confirm what should the signature look like if two supervisors will sign? if you can provide an example I am probably able to add this feature.
